{"url": "https://www.rei.com/",
 "crawl_time": "2025-06-20 21:41:50",
 "errors": "<twisted.python.failure.Failure twisted.internet.error.TimeoutError: User timeout caused connection failure: Getting https://www.rei.com/ took longer than 30.0 seconds..>"}
